pyccel/ast/core.py

```diff
@@ -4929,7 +4923,7 @@ def _sympystr(self, printer):
 
 
 
-class Slice(Basic):
+class Slice(Basic, PyccelOperator):
```

Comment:
@MenoIy I seem to have missed this in the review. Can you explain why you added this inheritance without moving `Slice` to `ast.operators.py` and without using any of the properties of `PyccelOperators` (notably it is important to call the `__init__` function)?

I wanted to move `Slice` in PR #637 but this inheritance stops me putting it where I was going to so I wanted to understand why it is necessary